### ISSUE TITLE
Straight panel corners?

### ISSUE THREAD

**Issue by Hotteterre** [theRealPadster/Mist-theme#13]
Hello, could I suggest to change the panel corners from round to transparent? I find this solution more consistent with the majority of GTK themes.

Specifically, I'd change the code from
```
.panel-corner {
    -panel-corner-radius: 0px;
    -panel-corner-background-color: transparent;
    -panel-corner-border-width: 1px;
    -panel-corner-border-color: transparent;
}
```
to
```
.panel-corner {
    -panel-corner-radius: 0px;
    -panel-corner-background-color: transparent;
    -panel-corner-border-width: 1px;
    -panel-corner-border-color: transparent;
}
```
Excuse me for the boring request :-) and thank you for the great work!

**Comment by Hotteterre** [comment]
I hope you aren't afraid of it: I'll put the modified code as a pull request.
Thank you!!!

**Comment by theRealPadster** [comment]
The theme has a transparent top bar, so the radius on the panel corners have no effect. That said, I can see the point in having nothing "there" if there is nothing visible, so I'll merge this in. Thanks for the interest :)

**Comment by theRealPadster** [comment]
Merged in 98a3bbb00ebf8599dd8fc7a0ddedc0618f1675e8

**Comment by Hotteterre** [comment]
I think it affects the shape of maximized windows. So their appearance seems to me more 'natural'. Thank you for the great theme and the kindness!
